src/pytest_mock_resources/patch/redshift/psycopg2.py

```diff
@@ -22,33 +28,32 @@ class CustomCursor(psycopg2.extensions.cursor):
         """A custom cursor class to define a custom execute method."""
 
         def execute(self, sql, args=None):
-            dsn_params = self.connection.get_dsn_parameters()
-            engine = create_engine(
```

Comment:
this was super suspect, there's no way we were getting anything like real default psycopg2 behavior doing it this way.